robber/matchers/equal.py

```diff
@@ -5,6 +5,13 @@
 from robber.helper import unicode_to_str
 from robber.matchers.base import Base
 
+try:
+    from collections import OrderedDict
+except ImportError:
+    ordered_dict_available = False
+else:
+    ordered_dict_available = True
```

Comment:
Are you sure that there's sometime ordered_dict is not available. I thought that any python env should come with it?